data/streetstylecoder.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name":"Harsh Dhariwal",
+  "displayStatsPublic":"true",
```

Comment:
We no longer need this `displayStatsPublic` key. So please, remove this from your `json`

```suggestion
```